### PR TITLE
Paul/perf mon

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206533398594286

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -18,6 +19,7 @@ import { NumberMap } from './types/types'
 import { log, logToServer } from './util/logger'
 import { initInfoServer } from './util/network'
 
+// LogBox.ignoreAllLogs()
```
Comment (swansontec) [inline src/app.ts:22]: Leftover comment?

File: package.json [hunk 2]
```diff
@@ -168,6 +169,7 @@
     "react-native-email-link": "^1.14.5",
     "react-native-fast-image": "^8.5.11",
     "react-native-fast-shadow": "^0.1.0",
+    "react-native-file-access": "^3.0.5",
```
Comment (swansontec) [inline package.json:172]: This seems unused?

**Review by swansontec — CHANGES_REQUESTED** [review]
I think there is an extra native dependency in here. Can we remove it?

File: src/app.ts (hunk 1 above)
Comment (paullinator) [reply src/app.ts:22]: Actually. I should have added a comment. I intentionally added that to easily disable the logbox.

File: package.json (hunk 2 above)
Comment (paullinator) [reply package.json:172]: It was asked to be added by bugsnag

https://docs.bugsnag.com/performance/integration-guides/react-native/

It appears to be imported extensively in their code although it's not a peer dependency.

**Review by swansontec — APPROVED** [review]

**Pull request merged**